osgi.specs/docbook/160/service.jakartaws.xml

```diff
@@ -239,6 +244,51 @@ public class SOAPLoggingHandler implements SOAPHandler&lt;SOAPMessageContext&gt;
             to also support publishing to the Http Whiteboard Service using property <emphasis>osgi.jakarta.xml.ws.endpoint.http.contextpath</emphasis> as described in <emphasis>Registering Endpoint Implementor</emphasis>
             </para>
         </section>
+        <section>
+        	<title>The WebserviceServiceRuntime</title>
+			 <para>The <xref
+			      linkend="org.osgi.service.webservice.runtime.WebserviceServiceRuntime" xrefstyle="hyperlink"/> service represents the runtime state information
+			      of a Jakarta XML Web Services Whiteboard instance. This information is provided through Data Transfer Objects (DTOs). The architecture of
+			      OSGi DTOs is described in <xref linkend="intro.core.release" xrefstyle="template:%t"/>.</para>
+			<para>The <code>WebserviceServiceRuntime</code> provides service registration properties to describe its underlying Jakarta XML Web
+      				Services Whiteboard. These service properties can include implementation-specific key-value pairs.
+      				They also include the following:</para>
+
+      <table pgwide="1">
+        <title>Service properties for the <code>WebserviceServiceRuntime</code> service</title>
+
+        <tgroup cols="3">
+          <colspec colnum="1" colwidth="1.9*"/>
+
+          <colspec colnum="2" colwidth="0.7*"/>
+
+          <colspec colnum="3" colwidth="4.2*"/>
+
+          <thead>
+            <row>
+              <entry>Service Property Name</entry>
+
+              <entry>Type</entry>
+
+              <entry>Description</entry>
+            </row>
+          </thead>
+
+          <tbody>
+            <row>
+              <entry><code>service.changecount</code></entry>
+              <entry><code>Long</code></entry>
+              <entry><para>Whenever the DTOs available from the WebserviceServiceRuntime service change, the value
+              of this property will increase.</para> <para>This allows
+              interested parties to be notified of changes to the DTOs by
+              observing Service Events of type <code>MODIFIED</code> for the
+              <xref linkend="org.osgi.service.webservice.runtime.WebserviceServiceRuntime"
+              xrefstyle="hyperlink"/> service. See <code>org.osgi.framework.Constants.SERVICE_CHANGECOUNT</code> in <xref linkend="intro.core.release" xrefstyle="template:%t"/>.</para></entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </table>
+        </section>
```

Comment:
This section should have a subsection describing the method for obtaining a DTO, and its rules:

- Is the DTO live or a snapshot (hopefully a snapshot!)
- What is stored in the returned DTO (endpoints, handlers, failures)

Reply:
Done, I updated the initial description with the current version of the spec.